app/templates/merchandDashboard.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
```

Comment:
Pourquoi n'a tu pas extends le fichier? Tous les fichiers html devrait etre basé sur le meme template directement ou indirectement

Reply:
Les pages ne sont pas toute faites. C'est in progress

Reply:
@komlan55 va voir la page `login.html` et `index.html`